src/main/java/org/alliancegenome/curation_api/jobs/executors/GeneExpressionExecutor.java

```diff
@@ -1,26 +1,34 @@
 package org.alliancegenome.curation_api.jobs.executors;
 
-import java.io.FileInputStream;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.zip.GZIPInputStream;
-
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
 import org.alliancegenome.curation_api.enums.BackendBulkDataProvider;
+import org.alliancegenome.curation_api.exceptions.ObjectUpdateException;
 import org.alliancegenome.curation_api.interfaces.AGRCurationSchemaVersion;
 import org.alliancegenome.curation_api.model.entities.GeneExpressionAnnotation;
+import org.alliancegenome.curation_api.model.entities.GeneExpressionExperiment;
 import org.alliancegenome.curation_api.model.entities.bulkloads.BulkFMSLoad;
 import org.alliancegenome.curation_api.model.entities.bulkloads.BulkLoadFileHistory;
 import org.alliancegenome.curation_api.model.ingest.dto.fms.GeneExpressionIngestFmsDTO;
 import org.alliancegenome.curation_api.services.GeneExpressionAnnotationService;
+import org.alliancegenome.curation_api.services.GeneExpressionExperimentService;
+import org.alliancegenome.curation_api.util.ProcessDisplayHelper;
 import org.apache.commons.lang3.StringUtils;
 
-import jakarta.enterprise.context.ApplicationScoped;
-import jakarta.inject.Inject;
+import java.io.FileInputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.zip.GZIPInputStream;
 
 @ApplicationScoped
 public class GeneExpressionExecutor extends LoadFileExecutor {
-	@Inject
-	GeneExpressionAnnotationService geneExpressionAnnotationService;
+	@Inject GeneExpressionAnnotationService geneExpressionAnnotationService;
+	@Inject GeneExpressionExperimentService geneExpressionExperimentService;
+	static final String ANNOTATIONS = "gen_exp_annotations";
```

Comment:
I still don't love this naming.  My preference would be just "Annotations" and "Experiments" for the counts.

Reply:
Renamed